### PR TITLE
update links in documentation

### PR DESCRIPTION
**Pull request by kodami**

### CODE REVIEW

**Comment by ghettovoice** [comment]
I'm using docsify that renders markdown files in the `docs` directory as simple docs site. You can see it here https://vuelayers.github.io/#/.  
Now relative links work for github.com but there is an issue with relative links in docsify https://github.com/docsifyjs/docsify/issues/424. Docsify treats all links as absolute path.  

I think the workaround would be to use absolute paths including `/docs` in files. Then I can replace the `/docs` part with custom docsify plugin on documentation site, so links will work everywhere.

Can you replace all relative links with absolute started from `/docs/`?

File: docs/component/feature.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 > Geographic feature representation
 
 `vl-feature` provides properties similar the **GeoJSON** feature.  It has one 
-**default slot** and acts like geometry container for [`vl-geom-*`](component/point-geom.md) 
+**default slot** and acts like geometry container for [`vl-geom-*`](point-geom.md)
```
Comment (ghettovoice) [inline docs/component/feature.md]: `point-geom.md` -> `/docs/component/point-geom.md`

File: docs/component/feature.md [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 
 `vl-feature` provides properties similar the **GeoJSON** feature.  It has one 
-**default slot** and acts like geometry container for [`vl-geom-*`](component/point-geom.md) 
+**default slot** and acts like geometry container for [`vl-geom-*`](point-geom.md) 
 components, without geometry component nothing will be rendered.
 
 `vl-feature` components can be placed inside component that are mixes in 
-[`featuresContainer`](mixin/features-container.md) mixin, such as [`vl-map`](component/map.md), 
-[`vl-source-vector`](component/vector-source.md) and etc.
+[`featuresContainer`](../mixin/features-container.md) mixin, such as [`vl-map`](map.md),
```
Comment (ghettovoice) [inline docs/component/feature.md]: `../mixin/features-container.md` -> `/docs/mixin/features-container.md`

**Comment by ghettovoice** [comment]
thanks anyway your contribution :+1:

**Comment by kodami** [comment]
ok... I edited files but I don't know if I did this correctly, please let me know what I did wrong. BTW should I change commit messages on edited files to something else?

**Comment by ghettovoice** [comment]
> I edited files but I don't know if I did this correctly

All right! 

>  should I change commit messages on edited files to something else?

It would be great to squash all commits to the one with message like this pull request has.

Thanks!

**Pull request merged**